.github/workflows/TICKET-MATE-BE-SCHEMA-VERIFY.yaml

```
@@ -0,0 +1,201 @@
+            sed -n '1,60p' "$FLY_SCHEMA"
+            echo "----- [마지막 30줄] -----"
+            tail -n 30 "$FLY_SCHEMA"
+          else
+            echo "❌ Flyway 스키마 파일을 찾을 수 없습니다: $FLY_SCHEMA"
+            echo "📂 현재 생성 디렉터리 목록:"
+            ls -la "$GEN_DIR" || true
+            exit 1
+          fi
+
+      - name: Hibernate DDL 생성 (스프링 미사용, 해당 테스트만 실행)
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:12: Double quote to prevent globbing and word splitting

(shellcheck)